packages/overlays/addon/components/drawer/index.hbs

```diff
@@ -25,14 +25,14 @@
     >
       {{#if this.showCloseButton}}
         <CloseButton
-          @onClick={{this.handleClose}}
+          @onClick={{@onClose}}
```

Comment:
Here I verified,  CloseButton component does the ``typeof === 'function'` check too